mono/utils/mono-merp.c

```diff
@@ -337,13 +337,11 @@ mono_merp_send (MERPStruct *merp)
 
 	// Only one we define on OSX
 	if (pid == 0) {
-		const char *open_path = "/usr/bin/open";
-		const char *argvOpen[] = {open_path, "-a", config.merpGUIPath, NULL};
-		execv (open_path, (char**)argvOpen);
+		const char *argvOpen[] = {config.merpGUIPath, NULL};
+		execv (config.merpGUIPath, (char**)argvOpen);
```

Comment:
I tried this in a standalone sample, and I don't think we can `execv` an app bundle directly.
So I think we should do:
```c
		const char *open_path = "/usr/bin/open";
		const char *argvOpen[] = {open_path, "-n", "-W", "-a", config.merpGUIPath, NULL};
		execv (open_path, (char**)argvOpen);
```

Reply:
The problem with this is that `open` does not seem to "pass through" the exit code of the target app